images/awx-ee/execution-environment.yml

```diff
@@ -11,7 +11,20 @@ additional_build_steps:
     - ARG NVM_INSTALL_VERSION=v0.39.1
     - ARG NVM_DIR=$HOME/.nvm
     - ARG NODE_VERSION=v14.15.1
-    - RUN dnf module enable php:7.4 -y
+    # Custom rpm for libsodium.
+    - |-
+      RUN dnf install --assumeyes wget && \
+          wget https://kojipkgs.fedoraproject.org//packages/libsodium/1.0.18/2.el8/x86_64/libsodium-1.0.18-2.el8.x86_64.rpm -O /tmp/libsodium.rpm && \
+          rpm -i /tmp/libsodium.rpm && \
+          rm -rf /tmp/*
+    # Use remi repo for PHP.
+    - |-
+      RUN dnf config-manager --set-enabled powertools && \
+          dnf install --assumeyes https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+          dnf install --assumeyes https://rpms.remirepo.net/enterprise/remi-release-8.rpm && \
+          dnf module reset --assumeyes php && \
```

Comment:
This sets up the image to use the remi package repo, which seems like the go for PHP.

Reply:
I've used remi for nearly every custom ansible build since around PHP 5.1 and I've never had any problems with it, so it should be solid for our purposes. CentOS is notoriously difficult to get latest packages which is something remi works around (RHEL prefers extremely stable releases rather than latest)